### ISSUE TITLE
Add CI + change location of documentation site

### ISSUE THREAD

**Issue by bsaul** [bsaul/smd#2]
* [x] add CI to build/test package
* [x] update location of documentation site (use github.io)
* [x] update any links to documentation site (e.g. `README.md`)

**Comment by ddsjoberg** [comment]
I can definitely do these tasks!

**Comment by ddsjoberg** [comment]
I submitted a PR to run the R CMD Checks and create a code coverage report via GitHub Actions.

The next part would be to update the pkgdown website location. I think you should probably do this one on your own, because it requires more access to the repo.

- Start by running `usethis::use_pkgdown_github_pages(). This will complete most of the steps for you--assuming you have GitHub PAT/token setup because some of these tasks require the use of the GitHub API. If you haven't yet set this up, here are the instructions: https://happygitwithr.com/https-pat#tldr
    - creates a pkgdown config file and adds relevant files or directories to .Rbuildignore and .gitignore.
    - prepares to publish the pkgdown site from the gh-pages branch (instead of serving the site from a docs folder).
    - configures a GitHub Action to automatically build the pkgdown site and deploy it via GitHub Pages
    - the pkgdown site's URL is added to the pkgdown configuration file, to the URL field of DESCRIPTION, and to the GitHub repo. 

I saw that you already have a `_pkgdown.yml` file, so the first step will likely be skipped or you'll be prompted if you wish to overwrite it. If you keep the one you have, I saw that you have a line `destination: docs`: you can delete this as you'll serve the site from the `gh-pages` branch.

You can also add a chunk like this to the `_pkgdown.yml` file, so your name as the author will be a hyperlink to your personal website.

```
authors:
  Bradley Saul:
    href: "https://www.functionalstatistics.com/"
```

Hope this helps!! Thanks again for the smd package! It's great!!

**Comment by bsaul** [comment]
Thanks for initializing the CI @ddsjoberg. I think the site has now been migrated and links fixed via #6 and #8.
